PyDSS/element_fields.py

```diff
@@ -0,0 +1,119 @@
+ELEMENT_FIELDS = {
+    "Lines": [
```

Comment:
Does this dict need an entry for Buses?

Reply:
Is class=Buses property=puVmagAngle something that people will want to export?

Reply:
Yes almost always
I will add that also

Reply:
done